src/components/RecordsSummary/RecordsSummary.js

```diff
@@ -0,0 +1,165 @@
+import React from 'react';
+import { shape, instanceOf, string } from 'prop-types';
+import { pick } from 'ramda';
+import { connect } from 'react-redux';
+import {
+  StyleSheet, Text, View,
+} from 'react-native';
+
+import {
+  getRecordsTotal,
+  getDataRange,
+} from '../../resources/fhirReader';
+import { supportedResourcesSelector } from '../../redux/selectors';
+import Colors from '../../constants/Colors';
+import { clearPatientData } from '../../features/patient/patientDataSlice';
+import RESOURCE_TYPES from '../../resources/resourceTypes';
+
+const ResourceTypeRow = ({ resourceType, resources, resourceIdsGroupedByType }) => {
+  const consolidatedIds = Object.values(resourceIdsGroupedByType[resourceType])
+    .reduce((acc, cur) => {
+      acc.push(...cur);
+      return acc;
+    }, []);
+  const inflatedObjects = pick(consolidatedIds, resources);
+  const [, latestDate] = getDataRange(inflatedObjects, 'Y');
+  return (
+    <View style={styles.resourceTypeRow}>
+      <Text style={styles.resourceName}>{RESOURCE_TYPES[resourceType]}</Text>
+      <Text style={styles.resourceCount}>{consolidatedIds.length}</Text>
+      <Text style={styles.resourceLatestDate}>{latestDate}</Text>
+    </View>
+  );
+};
+
+ResourceTypeRow.propTypes = {
+  resourceType: string.isRequired,
+  resources: instanceOf(Object).isRequired,
+  resourceIdsGroupedByType: instanceOf(Object).isRequired,
+};
+
+const RecordsSummary = ({
+  resourceIdsGroupedByType, resources,
+}) => {
+  const recordsTotal = getRecordsTotal(resources);
+
+  return (
+    <View style={styles.recordSummaryContainer}>
+      <View style={styles.recordsHeader}>
+        <Text style={styles.recordsHeaderText}>
+          Records
+        </Text>
+        <Text style={styles.recordsHeaderTotal}>
+          {recordsTotal}
+          {' '}
+          Total
```

Comment:
string interpolate: `${recordsTotal} Total`

Reply:
Thanks for the tip! I also had to wrap it in braces, so in the end ``{`${recordsTotal} Total`}`` did the job.

To do backticks in code you can start the entire thing with (backtick)(backtick) instead of the usual one (backtick) and then any backtics inside don't need escaping.